src/cluster.c

```diff
@@ -3693,7 +3765,7 @@ void clusterCron(void) {
             }
         }
         if (min_pong_node) {
-            serverLog(LL_DEBUG,"Pinging node %.40s", min_pong_node->name);
+            serverLog(LL_DEBUG,"Pinging node %.40s %s", min_pong_node->name, min_pong_node->human_readable_name);
```

Comment:
I think we should maybe do something like, "Pining node <whatever> (<human readable name if present>)", or something to indicate it's the name and not sure the word after.

Reply:
Looks like all of this is removed now.